### PR TITLE
chore: adding value prop to README

### PR DESCRIPTION
**Pull request by sharkparty**
**Before submitting a pull request,** please make sure the following is done:

- [x] Clone [the repository](https://github.com/retailmenot/anchor) and branch from `master` (`git checkout -b feature/button`).
- [x] Run `npm ci` in the repository root.
- [x] Use [conventional commits](https://www.conventionalcoamits.org/en/v1.0.0-beta.3/) for all work.
- [x] If you've fixed a bug or added code, [add stories](https://storybook.js.org/docs/basics/introduction/)!
- [x] If you've fixed a bug or added code that should be tested, [add tests](https://jestjs.io/docs/en/getting-started)!
- [x] Ensure the test suite passes & coverage thresholds are met (`npm test`).
- [x] Format your code with [prettier](https://github.com/prettier/prettier) (`npm run pretty`).
- [x] Make sure your code lints (`npm run lint`).

---------
**Outline your feature or bug-fix below**

Added docs & updated PR template.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -30,6 +32,14 @@ import { Menu } from '@retailmenot/anchor';
 
 [View our docs site for more usage guidelines](https://retailmenot.design/)
```
Comment (shaundub) [inline readme.md]: We have to link to anchor.retailmenot.com for now until we can work out naked domain routing.

File: readme.md [hunk 2]
```diff
@@ -30,6 +32,14 @@ import { Menu } from '@retailmenot/anchor';
...
 
+### Why Anchor?
+
+* Isomorphic - We spent a lot of time trying to write out a library that doesn't require a Webpack expert to render server-side
```
Comment (shaundub) [inline readme.md]: Did we succeed? Maybe switch to more active tone. We are an isomorphic-first library which doesn't require a Webpack expert...

Comment (sharkparty) [reply readme.md]: Good point. Plus this is shorter.

File: readme.md (hunk 1 above)
Comment (sharkparty) [reply readme.md]: I keep forgetting about this and will likely need a few more reminders.

Comment (shaundub) [reply readme.md]: I got you :)

**Comment by shaundub** [comment]
👍

File: readme.md (hunk 1 above)
Comment (shaundub) [reply readme.md]: Correction, this should be https://anchor.retailmenot.design

**Pull request merged**